### PR TITLE
Update online unsafe recovery doc

### PR DESCRIPTION
**Pull request by Connor1996**
### What is changed, added or deleted? (Required)

Update online unsafe recovery doc to adapt to the latest output

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/tikv/tikv/issues/10483

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by v01dstar — APPROVED** [review]
LGTM

File: pd-control.md [hunk 1]
```diff
@@ -1352,27 +1351,6 @@ Success!
...
-  "Store 8, ..."
-  "...",
-  "Execution Progress:",
-  "Store 10 finished,",
-  "Store 7 not yet finished",
-  "...",
-]
-```
-
 ## jq 格式化 json 输出示例
```
Comment (Oreoxmt) [inline pd-control.md:1354]:
```suggestion
## jq 格式化 JSON 输出示例
```

File: tikv-control.md [hunk 2]
```diff
@@ -487,7 +487,11 @@ tikv-ctl --host ip:port modify-tikv-config -n rocksdb.rate-bytes-per-sec -v "1GB
 success
 ```
 
-### 强制 Region 从多副本失败状态恢复服务（慎用）
+### 强制 Region 从多副本失败状态恢复服务（弃用）
+
+> **注意：**
+> 
+> 不推荐使用该功能，恢复需求可通过 `pd-ctl` 的 Online Unsafe Recovery 功能实现。它提供了一键式自动恢复的能力，无需停止服务等额外操作，具体使用方式请参考 [Online Unsafe Recovery 使用文档](./online-unsafe-recovery.md)。
```
Comment (Connor1996) [inline tikv-control.md:494]:
```suggestion
> 不推荐使用该功能，恢复需求可通过 `pd-ctl` 的 Online Unsafe Recovery 功能实现。它提供了一键式自动恢复的能力，无需停止服务等额外操作，具体使用方式请参考 [Online Unsafe Recovery 使用文档](/online-unsafe-recovery.md)。
```

File: online-unsafe-recovery.md [hunk 3]
```diff
@@ -38,45 +37,101 @@ Online Unsafe Recovery 功能适用于以下场景：
...
 
-### 第 2 步：移除无法自动恢复的节点
-
-使用 PD Control 执行 [`unsafe remove-failed-stores <store_id>[,<store_id>,...]`](/pd-control.md#unsafe-remove-failed-stores-store-ids--show--history)命令，移除无法自动恢复的节点。
+可通过 `--timeout <seconds>` 指定可允许执行恢复的最长时间。若未指定，默认为 5 分钟。当超时后，恢复中断报错。
 
 > **注意：**
 >
-> 此命令成功返回，仅表示请求已被接受，而不代表恢复成功。节点实际会在后台进行恢复。
+> - 由于此命令需要收集来自所有 Peer 的信息，可能会造成 PD 短时间内有明显的内存使用量上涨(100k 个 Peer 预计使用约 500MB 内存)。
```
Comment (Oreoxmt) [inline online-unsafe-recovery.md:54]:
```suggestion
> - 由于此命令需要收集来自所有 Peer 的信息，可能会造成 PD 短时间内有明显的内存使用量上涨（100k 个 Peer 预计使用约 500MiB 内存）。
```

**Review by Oreoxmt — APPROVED** [review]

File: online-unsafe-recovery.md [hunk 4]
```diff
@@ -38,45 +37,101 @@ Online Unsafe Recovery 功能适用于以下场景：
 * 离线节点导致部分数据确实不可用。
 * 离线节点确实无法自动恢复或重启。
 
-### 第 1 步：关闭各类调度
+### 第 1 步：移除无法自动恢复的节点
 
-暂时关闭负载均衡等各类内部调度。关闭后，建议等待约 10 分钟，使已经触发的调度能有充分的时间完成调度任务。
+使用 PD Control 执行 [`unsafe remove-failed-stores <store_id>[,<store_id>,...]`](/pd-control.md#unsafe-remove-failed-stores-store-ids--show) 命令，指定已确定无法恢复的 TiKV 节点，触发自动恢复。
```
Comment (TomShawn) [inline online-unsafe-recovery.md:42]:
```suggestion
使用 PD Control 执行 [`unsafe remove-failed-stores <store_id>[,<store_id>,...]`](/pd-control.md#unsafe-remove-failed-stores-store-ids--show) 命令，指定已确定无法恢复的 TiKV 节点，并触发自动恢复。
```

File: online-unsafe-recovery.md [hunk 5]
```diff
@@ -38,45 +37,101 @@ Online Unsafe Recovery 功能适用于以下场景：
...
 
-### 第 2 步：移除无法自动恢复的节点
-
-使用 PD Control 执行 [`unsafe remove-failed-stores <store_id>[,<store_id>,...]`](/pd-control.md#unsafe-remove-failed-stores-store-ids--show--history)命令，移除无法自动恢复的节点。
+可通过 `--timeout <seconds>` 指定可允许执行恢复的最长时间。若未指定，默认为 5 分钟。当超时后，恢复中断报错。
 
 > **注意：**
 >
-> 此命令成功返回，仅表示请求已被接受，而不代表恢复成功。节点实际会在后台进行恢复。
+> - 由于此命令需要收集来自所有 Peer 的信息，可能会造成 PD 短时间内有明显的内存使用量上涨（100k 个 Peer 预计使用约 500MiB 内存）。
```
Comment (TomShawn) [inline online-unsafe-recovery.md:54]: 100K 是指十万吗？
```suggestion
> - 由于此命令需要收集来自所有 Peer 的信息，可能会造成 PD 短时间内有明显的内存使用量上涨（100k 个 Peer 预计使用约 500 MiB 内存）。
```

File: online-unsafe-recovery.md [hunk 6]
```diff
@@ -38,45 +37,101 @@ Online Unsafe Recovery 功能适用于以下场景：
...
+> - 若执行过程中 PD 发生重启，需重新触发命令。
+> - 一旦执行，所指定的节点将被设为 Tombstone 状态，不再允许启动。
```
Comment (TomShawn) [inline online-unsafe-recovery.md:56]:
```suggestion
> - 一旦执行该命令，所指定的节点将被设为 Tombstone 状态，不再允许启动。
```

File: tikv-control.md [hunk 7]
```diff
@@ -487,7 +487,11 @@ tikv-ctl --host ip:port modify-tikv-config -n rocksdb.rate-bytes-per-sec -v "1GB
 success
 ```
 
-### 强制 Region 从多副本失败状态恢复服务（慎用）
+### 强制 Region 从多副本失败状态恢复服务（弃用）
+
+> **注意：**
```
Comment (TomShawn) [inline tikv-control.md:492]:
```suggestion
> **警告：**
```

File: online-unsafe-recovery.md [hunk 8]
```diff
@@ -38,45 +37,101 @@ Online Unsafe Recovery 功能适用于以下场景：
...
+```json
+{
+    "info": "Unsafe recovery failed: <error>",
+    "time": "......"
+}
 ```
 
+### 第 3 步： 检查数据索引一致性
+
+当恢复完成后，可以尝试运行一些简单 SQL 查询或写入操作确保数据可以读写。执行完成后，可能会导致数据索引不一致，请使用 SQL 的 `ADMIN CHECK`、`ADMIN RECOVER`、`ADMIN CLEANUP` 命令进行数据索引的一致性检查及恢复。
```
Comment (TomShawn) [inline online-unsafe-recovery.md:133]:
```suggestion
当恢复完成后，你可以尝试执行一些简单 SQL 查询或写入操作确保数据可以读写。执行完成后，可能会导致数据索引不一致，请使用 `ADMIN CHECK`、`ADMIN RECOVER`、`ADMIN CLEANUP` 命令进行数据索引的一致性检查及恢复。
```

File: online-unsafe-recovery.md [hunk 9]
```diff
@@ -38,45 +37,101 @@ Online Unsafe Recovery 功能适用于以下场景：
...
-> 此命令成功返回，仅表示请求已被接受，而不代表恢复成功。节点实际会在后台进行恢复。
+> - 由于此命令需要收集来自所有 Peer 的信息，可能会造成 PD 短时间内有明显的内存使用量上涨（100k 个 Peer 预计使用约 500MiB 内存）。
+> - 若执行过程中 PD 发生重启，需重新触发命令。
+> - 一旦执行，所指定的节点将被设为 Tombstone 状态，不再允许启动。
+
+### 第 2 步：查看进度等待结束
+
+节点移除命令运行成功后，使用 PD Control 执行 [`unsafe remove-failed-stores show`](/pd-control.md#config-show--set-option-value--placement-rules) 命令，查看移除进度。
+
+恢复过程分为多个阶段，按照 JSON 格式输出，每一阶段包括信息，时间，以及具体的恢复计划。例如：
```
Comment (TomShawn) [inline online-unsafe-recovery.md:62]:
```suggestion
恢复过程分为多个阶段，每个阶段的进度都以 JSON 格式输出。每一阶段的输出内容包括信息、时间，以及具体的恢复计划。例如：
```

**Review by TomShawn — COMMENTED** [review]
All minor comments

**Review by TomShawn — APPROVED** [review]

**Pull request merged**